packages/smooth_app/lib/pages/navigator/app_navigator.dart

```diff
@@ -282,6 +287,8 @@ class _SmoothGoRouter {
             }
           } else if (path == _ExternalRoutes.MOBILE_APP_DOWNLOAD) {
             return AppRoutes.HOME;
+          } else if (path == _ExternalRoutes.FORGOT_PASSWORD) {
```

Comment:
Could you move this statement before or after `SIGNUP`?
It will allow the regrouping of user management routes.
Thanks